Source/WebCore/page/scrolling/ScrollingStateScrollingNode.h

```diff
@@ -132,6 +142,8 @@ class ScrollingStateScrollingNode : public ScrollingStateNode {
     LayerRepresentation m_scrolledContentsLayer;
     LayerRepresentation m_horizontalScrollbarLayer;
     LayerRepresentation m_verticalScrollbarLayer;
+    
+    ScrollbarHoverState m_scrollbarHoverState { false, false };
```

Comment:
No need for the explicit initialization.

Reply:
Fixed.